### PR TITLE
Update partnership to have partner relation

### PR DESCRIPTION
**Pull request by zkhin**
Fixes: https://github.com/SeedCompany/cord-api-v3/issues/1196

### CODE REVIEW

File: src/components/partnership/partnership.service.ts [hunk 1]
```diff
@@ -76,6 +77,7 @@ export class PartnershipService {
     private readonly orgService: OrganizationService,
```
Comment (CarsonF) [inline src/components/partnership/partnership.service.ts:77]: OrgService isn't needed anymore

File: src/components/partnership/partnership.resolver.ts [hunk 2]
```diff
@@ -69,15 +72,14 @@ export class PartnershipResolver {
     return await this.files.resolveDefinedFile(partnership.agreement, session);
   }
 
-  //TODO: implement resolver to read child organization if it won't break the list reads
-  @ResolveField(() => Organization)
-  async organization(
+  @ResolveField(() => Partner)
+  async partner(
```
Comment (CarsonF) [inline src/components/partnership/partnership.resolver.ts:76]: I wonder if this should be secured now...

**Comment by CarsonF** [comment]
I'm going to wait to merge this until the UI is ready for it since it's a breaking change.

**Comment by zkhin** [comment]
> I'm going to wait to merge this until the UI is ready for it since it's a breaking change.

Okay

**Review by CarsonF — DISMISSED** [review]

**Review by CarsonF — APPROVED** [review]

**Pull request merged**